fluent-plugin-sd-dns.gemspec

```diff
@@ -0,0 +1,25 @@
+# encoding: utf-8
+$:.push File.expand_path('../lib', __FILE__)
+
+Gem::Specification.new do |gem|
+  gem.name        = "fluent-plugin-sd-dns"
+  gem.description = "DNS based service discovery plugin for Fluentd"
+  gem.license     = "Apache-2.0"
+  gem.homepage    = "https://github.com/fluent/fluent-plugin-sd-dns"
+  gem.summary     = gem.description
+  gem.version     = "0.1.0"
+  gem.authors     = ["Masahiro Nakagawa"]
+  gem.email       = "repeatedly@gmail.com"
+  #gem.platform    = Gem::Platform::RUBY
+  gem.files       = `git ls-files`.split("\n")
+  gem.test_files  = `git ls-files -- {test,spec,features}/*`.split("\n")
+  gem.executables = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
+  gem.require_paths = ['lib']
+
+  gem.add_dependency "fluentd", ">= 1"
```

Comment:
Fluentd v1.8.0 or later is required?
https://github.com/fluent/fluentd/commit/17ae60accda16a2f133fa95ee655428964c5ea0d